.github/workflows/ci.yaml

```diff
@@ -313,7 +313,7 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: scbexperimental/nmap
+          repository: securecodebox/nmap
```

Comment:
The image names for scanners would also have to be changed here.
I'm personally not really a big fan of this renaming. Would prefer to keep it short for the scanner. As these image can also be used without the secureCodeBox.

Reply:
I would prefer to prefix everything with the job that they are doing (hooks-<hookname>, scanner-<scannername>, parser-<scannername>) like with the parser images. What is your opinion on this @rseedorff ?